src/fugw/mappings/utils.py

```diff
@@ -1,3 +1,8 @@
+from copy import deepcopy
+
```

Comment:
It's meant to separate imports from standard and third-party libraries (you can check [PEP recommandations about this topic](https://peps.python.org/pep-0008/#imports)).
However, we could have our imports sorted automatically by [`isort`](https://pycqa.github.io/isort/) in a pre-commit hook so that we never have to discuss these things again :blush: